Control/Newtype/Generics.hs

```diff
@@ -365,9 +365,3 @@ instance Newtype (WrappedMonoid m) where
   type O (WrappedMonoid m) = m
   pack = WrapMonoid
   unpack (WrapMonoid m) = m
-
--- | @since 0.5.1
-instance Newtype (Option a) where
-  type O (Option a) = Maybe a
-  pack = Option
-  unpack (Option x) = x
```

Comment:
This instance should simply be hidden for GHC >= 9.2 with a bit of CPP.

Reply:
Done.